Sources/Crust/Parser.swift

```diff
@@ -206,6 +214,9 @@ extension Parser {
   }
 
   func parseModule() throws -> ModuleDeclSyntax {
+    guard currentToken?.tokenKind == .moduleKeyword else {
```

Comment:
You'll want to do this in `parseTopLevelModule`

Reply:
Thank you 👍 I was concerned about this but am just starting to familiarize myself with BNF.